basyx/aas/model/base.py

```diff
@@ -777,7 +777,7 @@ class Reference(metaclass=abc.ABCMeta):
     A reference is an ordered list of keys, each key referencing an element. The complete list of keys may for
     example be concatenated to a path that then gives unique access to an element or entity.
 
-    This is the abstract superclass of GlobalReference and ModelReference, which implements common attributes and
+    This is the abstract superclass of External and ModelReference, which implements common attributes and
```

Comment:
This `External` should be changed to `ExternalReference`, as it references the class.

Reply:
Oh, I introduced this change :joy: